src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -304,22 +306,40 @@ private void handleInvalidFileError(
             null
         );
 
-        ErrorMapping
-            .getFor(cause.getClass())
-            .ifPresent(errorCode -> {
-                try {
-                    sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, cause);
-                } catch (Exception exc) {
-                    log.error(
-                        "Error sending notification to the queue."
-                            + "File name: " + zipFilename + " "
-                            + "Container: " + containerName,
-                        exc
-                    );
-                }
-            });
+        Optionals.ifPresentOrElse(
+            ErrorMapping.getFor(cause.getClass()),
+            (errorCode) -> {
+                sendErrorMessage(
+                    zipFilename,
+                    containerName,
+                    cause, eventId,
+                    leaseId,
+                    errorCode
+                );
+                blobManager.tryMoveFileToRejectedContainer(zipFilename, containerName, leaseId);
+            },
+            () -> {
+                throw new ConfigurationException("Error code mapping not found for " + cause.getClass().getName());
+            }
+        );
+    }
 
-        blobManager.tryMoveFileToRejectedContainer(zipFilename, containerName, leaseId);
+    private void sendErrorMessage(
+        String zipFilename,
+        String containerName,
+        Exception cause,
+        Long eventId,
+        String leaseId,
+        ErrorCode errorCode
+    ) {
+        try {
+            sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, cause);
+        } catch (Exception exc) {
+            final String msg = "Error sending notification to the queue."
```

Comment:
Sorry to be requesting so many changes for this PR. It would be best for this message to be sth like `Error sending error notification...`, because we have notification by which we understand the ones we send to the orchestrator.

Reply:
No worries at all.